packages/app/src/server/routes/apiv3/pages.js

```diff
@@ -366,16 +366,17 @@ module.exports = (crowi) => {
   router.get('/recent', accessTokenParser, loginRequired, async(req, res) => {
     const limit = 20;
     const offset = parseInt(req.query.offset) || 0;
-
+    const page = parseInt(req.query.page) || 1;
+    // const offset = (+page - 1) * limit;
     const queryOptions = {
       offset,
       limit,
       includeTrashed: false,
       isRegExpEscapedFromPath: true,
       sort: 'updatedAt',
       desc: -1,
+      page,
```

Comment:
Remove `page` query and specify the same information with `offset` and `limit`.

Reply:
`page` query has been removed from pages api route